### PR TITLE
updated scss to reduce card padding to 100VW and fixed home to 100VW

### PR DESCRIPTION
**Pull request by SuwannaTruss**
https://trello.com/c/WsF9zNpQ/37-reduce-card-padding-to-not-go-beyond-100vw

updated in card-info component

image
image

### CODE REVIEW

File: WhaleSpotting/react-app/src/styles/Home.scss [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-    font-family: $fontFamily;
-  
+
     .home-contents {
+        display: flex;
+        flex-direction: column;
         margin: auto;
         max-width: $bodyWidth;
-        width: 100%;
+        // width: 100%;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Home.scss]: Commented code

File: WhaleSpotting/react-app/src/styles/Card.scss [hunk 2]
```diff
@@ -4,11 +4,11 @@
     border-radius: 10px;
     background-color: $primaryCard;
     margin: 15px 0;
+    box-sizing:border-box;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Card.scss]: No space

File: WhaleSpotting/react-app/src/styles/Navbar.scss [hunk 3]
```diff
@@ -10,6 +10,7 @@
     align-items: center;
     height: $headerHeight;
 
+
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Navbar.scss]: New line

File: WhaleSpotting/react-app/src/styles/Navbar.scss [hunk 4]
```diff
@@ -95,6 +96,7 @@
     @media(max-width: $mobileScreenSize) {
 
         justify-content: left;
+        padding-left: 5px;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Navbar.scss]: Was this purposeful?

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Could you also fix the report sighting button on the home page?

File: WhaleSpotting/react-app/src/styles/Navbar.scss (hunk 4 above)
Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Navbar.scss]: We think we need it. This allows space from the left edge of the screen.

With padding-left:
image

Without padding-left:
image

File: WhaleSpotting/react-app/src/styles/Home.scss (hunk 1 above)
Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Home.scss]: done

File: WhaleSpotting/react-app/src/styles/Card.scss (hunk 2 above)
Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Card.scss]: Done.

File: WhaleSpotting/react-app/src/styles/Navbar.scss (hunk 3 above)
Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Navbar.scss]: Done.

File: WhaleSpotting/react-app/src/styles/Navbar.scss (hunk 4 above)
Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Navbar.scss]: as per your comment, remove padding here, added margin on the current page. Done.

File: WhaleSpotting/react-app/src/styles/Card.scss (hunk 2 above)
Comment (oskarwilliams) [reply WhaleSpotting/react-app/src/styles/Card.scss]: Is this needed?

Comment (SuwannaTruss) [reply WhaleSpotting/react-app/src/styles/Card.scss]: Not really. It has been removed.

File: WhaleSpotting/react-app/src/styles/Home.scss [hunk 5]
```diff
@@ -3,15 +3,14 @@
...
     padding-bottom: $footerHeight;
     margin-top: $headerHeight;
-    display: flex;
-    flex-direction: column;
-    font-family: $fontFamily;
-  
+
     .home-contents {
+        display: flex;
+        flex-direction: column;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Home.scss]: Don't think this does anything

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Will update with my changes and then do this :)

File: WhaleSpotting/react-app/src/styles/Card.scss [hunk 6]
```diff
@@ -8,13 +8,17 @@
...
     }
 
     .first-column {
         width: 50%;
     }
 
+    .second-column {
+        width: 50%;
+    }
+
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Card.scss]: We don't need this :)

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**